src/integration-test/resources/CreateDisturbanceTest/__files/test2_createDisturbanceWhenSubscriptionExists/mappings/api-messaging-messages.json

```diff
@@ -1,99 +1,81 @@
-	"request": {
-		"urlPath": "/api-messaging/messages",
-		"method": "POST",
-		"queryParameters": {
-			"async": {
-				"equalTo": "true"
-			}
-		},
-		"bodyPatterns": [
+  "name": "api-messaging",
+  "request": {
+	"urlPath": "/api-messaging/messages",
+	"method": "POST",
+	"queryParameters": {
+	  "async": {
+		"equalTo": "true"
+	  }
+	},
+	"bodyPatterns": [
+	  {
+		"equalToJson": {
+		  "messages": [
+			{
+			  "party": {
+				"partyId": "fbfbd90c-4c47-11ec-81d3-0242ac130003"
+			  },
+			  "filters": {
+				"FACILITY_ID": [
+				  "facility-11"
+				],
+				"CATEGORY": [
+				  "COMMUNICATION"
+				],
+				"TYPE": [
+				  "DISTURBANCE"
+				]
+			  },
+			  "sender": {
+				"email": {
+				  "name": "[TEST] ServaNet",
+				  "address": "test@sundsvall.se"
+				},
+				"sms": {
+				  "name": "[TEST] ServaNet"
+				}
+			  },
+			  "subject": "[TEST] Driftstörning på Streetname 111",
+			  "message": "[TEST] Hej! Just nu har vi en driftstörning i fibernätet på Streetname 111 som vi felsöker. Hälsningar, ServaNet"
+			},
 			{
-				"equalToJson": {
-					"messages": [
-						{
-							"party": {
-								"partyId": "fbfbd90c-4c47-11ec-81d3-0242ac130003"
-							},
-							"headers": [
-								{
-									"name": "TYPE",
-									"values": [
-										"DISTURBANCE"
-									]
-								},
-								{
-									"name": "FACILITY_ID",
-									"values": [
-										"facility-11"
-									]
-								},
-								{
-									"name": "CATEGORY",
-									"values": [
-										"COMMUNICATION"
-									]
-								}
-							],
-							"sender": {
-								"sms": {
-									"name": "[TEST] ServaNet"
-								},
-								"email": {
-									"name": "[TEST] ServaNet",
-									"address": "test@sundsvall.se"
-								}
-							},
-							"subject": "[TEST] Driftstörning på Streetname 111",
-							"message": "[TEST] Hej! Just nu har vi en driftstörning i fibernätet på Streetname 111 som vi felsöker. Hälsningar, ServaNet"
-						},
-						{
-							"party": {
-								"partyId": "257f6aa0-4c48-11ec-81d3-0242ac130003"
-							},
-							"headers": [
-								{
-									"name": "TYPE",
-									"values": [
-										"DISTURBANCE"
-									]
-								},
-								{
-									"name": "FACILITY_ID",
-									"values": [
-										"facility-33"
-									]
-								},
-								{
-									"name": "CATEGORY",
-									"values": [
-										"COMMUNICATION"
-									]
-								}
-							],
-							"sender": {
-								"sms": {
-									"name": "[TEST] ServaNet"
-								},
-								"email": {
-									"name": "[TEST] ServaNet",
-									"address": "test@sundsvall.se"
-								}
-							},
-							"subject": "[TEST] Driftstörning på Streetname 333",
-							"message": "[TEST] Hej! Just nu har vi en driftstörning i fibernätet på Streetname 333 som vi felsöker. Hälsningar, ServaNet"
-						}
-					]
+			  "party": {
+				"partyId": "257f6aa0-4c48-11ec-81d3-0242ac130003"
+			  },
+			  "filters": {
+				"FACILITY_ID": [
```

Comment:
I think that we should send the keys in lower-camelCase (e.g. "facilityId", "category", etc).
That's what contactSettings are expecting.

Reply:
Didn't know, I'll fix